distribution/lib/Standard/Database/0.0.0-dev/src/Internal/Postgres/Postgres_Dialect.enso

```diff
@@ -264,6 +270,13 @@ concat_ops =
     concat = Base_Generator.make_concat make_raw_concat_expr make_contains_expr
     [["CONCAT", concat (has_quote=False)], ["CONCAT_QUOTE_IF_NEEDED", concat (has_quote=True)]]
 
+## PRIVATE
+trim_ops =
+    make_fn fn_name = Base_Generator.lift_binary_op fn_name input-> chars-> case chars of
+            Nothing -> Builder.code fn_name+"(" ++ input ++ ")"
+            _ : Text -> Builder.code fn_name+"(" ++ input ++ ", " ++ (Builder.interpolation chars) ++ ")"
```

Comment:
Are you sure this `_ : Text` branch is ever taken?? I'm surprised for it to be here.
How I remember the generator architecture is - the operations get as arguments _already built_ subexpressions - so if there was a `Constant (_ : Text)` there it should already be converted into an interpolation.

Reply:
you're right - the lift does it already.